CodeEditModules/Modules/WorkspaceClient/src/Live.swift

```diff
@@ -222,3 +180,26 @@ public extension WorkspaceClient {
         )
     }
 }
+
+extension URL {
+    var attributes: [FileAttributeKey: Any]? {
+        do {
+            return try FileManager.default.attributesOfItem(atPath: path)
+        } catch let error as NSError {
+            print("FileAttribute error: \(error)")
+        }
+        return nil
+    }
+
+    var fileSize: UInt64 {
+        return attributes?[.size] as? UInt64 ?? UInt64(0)
+    }
+
+    var fileSizeString: String {
+        return ByteCountFormatter.string(fromByteCount: Int64(fileSize), countStyle: .file)
+    }
+
+    var creationDate: Date? {
+        return attributes?[.creationDate] as? Date
+    }
+}
```

Comment:
It seems to me that this code isn't used anywhere. Are we sure we need to add it in this PR?

Reply:
it used to be used in a file priority thing that was removed in favour of only loading visible items. I removed it.